packages/lpa-questionnaire-web-app/src/controllers/contact-us.js

```diff
@@ -0,0 +1,13 @@
+const { VIEW } = require('../lib/views');
+const { renderView } = require('../util/render');
+
+async function renderContactUs(req, res) {
+  return renderView(res, VIEW.CONTACT_US, {
+    prefix: 'appeal-questionnaire',
+    backLink: req?.session?.backLink || `/${req.params.id}/${VIEW.TASK_LIST}`,
```

Comment:
There is a unit test that covers the second part of this conditional (ie: when `req.session.backLink` is not set but there doesn't appear to be a test to cover the first part of the conditional (ie: when `req.session.backLink` has a value)?